server/lib/notifications.js

```diff
@@ -137,6 +141,61 @@ export async function notifyAdminsOfCollective(CollectiveId, activity, options =
   return notifySubscribers(adminUsers, activity, options);
 }
 
+async function notifyAdminsOfCollectiveAndCheckUserExpensesAmount(activity) {
+  try {
+    await notifyAdminsOfCollective(activity.data.collective.id, activity);
+    await notifyUserIfExpensesExceededMinimumFormAmount(activity);  
+  } catch (error) {
+    console.log(error);
```

Comment:
Is there a plan to handle this error somewhere? Is there a way to notify the developers of an error?

Reply:
I consoled log instead of throwing an error in this case because this method is executed asynchronously. But that's something I noted we need to improve: error handling, we need to do `trycatch` is most async await functions and then either throw an error or at least log it properly( using `debug` or a log library). For now I can use the `debug` library instead of console.log as we're using in other places as well, but that's definitely something we can agree on always take the same actions.

Reply:
there are plenty of places throughout the code where we don't do much to handle those errors.
What would you recommend @HipsterBrown?

We have emails being sent to the server-errors email group but it's a bit overwhelming and it hasn't been super helpful in my view. What would be better here?

Reply:
What I used to do in the past is one or 2 actions:

- log the error using a log library(love [pino](https://github.com/pinojs/pino) )
- Warn devs through some third party modules (Only if it was necessary)